flict/__main__.py

```diff
@@ -49,6 +49,7 @@
 PROGRAM_COPYRIGHT = "(c) 2021 Henrik Sandklef<hesa@sandklef.com>"
 PROGRAM_LICENSE = "GPL-3.0-or-later"
 PROGRAM_AUTHOR = "Henrik Sandklef"
+PROGRAM_ATTRIBUTION = "flict is using the license compatibility matrix from osadl.org,\n  which can be found at https://www.osadl.org/fileadmin/checklists/matrix.html"
```

Comment:
to be super picky... the space after the line break and before `which` isn't needed

Reply:
There's two spaces 
They will match the indent by argparse:

```

COPYRIGHT
  Copyright (c) 2021 Henrik Sandklef<hesa@sandklef.com>.
  License GPL-3.0-or-later

ATTRIBUTION
  flict is using the license compatibility matrix from osadl.org,
  which can be found at https://www.osadl.org/fileadmin/checklists/matrix.html

SEE ALSO
```

Reply:
... which also displays a need to update the (c) notice all over :)

Reply:
I see, I will officially label you a console artist :-)

Reply:
Yes!!! I've seen Iggy Pop live and now this. I am complete.